src/projects/diff/group.html

```diff
@@ -303,6 +320,17 @@ <h2 hidden$="[[editable]]">[[displayTitle]]</h2>
       _resetCollapsed: function() {
         this.collapsed = false;
         this.collapseComments = true;
+      },
+      _updateComments: function(groupComments, group) {
+        if(group){
+          var newComments = [];
+          for(var i = 0;i< groupComments.base.length; i++){
+            // if(groupComments.base[i].groupID === group.id){
```

Comment:
This line needs to be commented out, because of the non sync ordering.
The group ID changes every reload.